src/apps/menu/menu.tsx

```diff
@@ -51,8 +51,8 @@ const Menu: FC<MenuProps> = ({ pageSize }) => {
         type="button"
         aria-label={
           userData
-            ? t("menuUserIconAriaLabelText")
-            : t("menuUserIconAriaLabelLoggedOutText")
+            ? t("menuUserIconAriaLabelLoggedOutText")
+            : t("menuUserIconAriaLabelText")
```

Comment:
I might be doing something wrong, but I am seeing this when I am logged in:

```
<button class="header__menu-profile header__button btn-ui" type="button" aria-label="Open login menu" tabindex="0">
  <img src="..." alt="">
  <span class="text-small-caption">
    Testkort DDF 07/Tøstesen
  </span>
</button>
```

Reply:
I fixed it according to what we talked about. Much confused.